### ISSUE TITLE
[FR] Overall Dashboard for easy viewing

### ISSUE THREAD

**Issue by sasuke00** [mlflow/mlflow#19508]
### Willingness to contribute

Yes. I would be willing to contribute this feature with guidance from the MLflow community.

### Proposal Summary

Have a centralized dashboard to view the latest progress of current running training which is easy to track the status and resources, and then able to show the model performance

### Motivation

> #### What is the use case for this feature?
- For collaboration between team member and easy for them to track the resources

> #### Why is this use case valuable to support for MLflow users in general?
- Have a centralized dashboard can view all important data at once instead of checking one by one

> #### Why is this use case valuable to support for your project(s) or organization?
-  Lack of useful monitoring which will lead to insufficient resources sometimes that can cause the program error

> #### Why is it currently difficult to achieve this use case?
- Currently there is no dashboard to view the status of the training which is active and model performance, we have to use compare to view it which is not convenient

### What machine learning domain(s) is this feature request about?

- [ ] `domain/genai`: LLMs, Agents, and other GenAI-related use cases
- [ ] `domain/classical-ml`: Traditional machine learning, such as linear regression.
- [ ] `domain/deep-learning`: Deep learning and neural networks.
- [x] `domain/platform`: MLflow platform foundation, not specific to a particular machine learning domain.

### What area(s) of MLflow is this feature request about?

- [x] `area/tracking`: Tracking Service, tracking client APIs, autologging
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/scoring`: MLflow model serving, deployment tools, Spark UDFs
- [ ] `area/evaluation`: MLflow model evaluation features, evaluation metrics, and evaluation workflows
- [ ] `area/prompt`: MLflow prompt engineering features, prompt templates, and prompt management
- [ ] `area/tracing`: MLflow Tracing features, tracing APIs, and LLM tracing functionality
- [ ] `area/gateway`: MLflow AI Gateway client APIs, server, and third-party integrations
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/uiux`: Front-end, user experience, plotting
- [ ] `area/docs`: MLflow documentation pages

**Comment by WeichenXu123** [comment]
For an created autologging run, 

You can track the training status by checking the loss metrics chart and the system metrics chart for resources utilizations.

and the runs list in an experiment is similar to a centralized view of all runs.

could you elaborate your idea and upload some draft UI image for describing your concept, maybe I haven't fully got your point.

**Comment by sasuke00** [comment]
It would be better to have a centralized dashboard which directly show all the current active models that are in the progress of traning instead of I need to manually to trigger the checking
